charts/clearblade-monitoring/charts/cb-monitoring/templates/monitoring-config.yaml

```diff
@@ -11,48 +11,50 @@ data:
    groups:
    - name: Alerts
      rules:
-      {{- if .Values.alerts.containerRestart }}
-     - alert: ContainerRestart
-       expr: time() - container_start_time_seconds{container=~"(clearblade|cb-.*)",container!~".*exporter",namespace="{{ $.Values.global.deploymentnamespace }}"} < 300
+      {{- range $n := .Values.global.deploymentNamespaces }}
+      {{- if $.Values.alerts.containerRestart }}
+     - alert: {{$n.name}}ContainerRestart
+       expr: time() - container_start_time_seconds{container=~"(clearblade|cb-.*)",container!~".*exporter",namespace="{{ $n.name }}"} < 300
        for: 1m
        annotations:
          summary: "{{ "{{" }} $labels.container }} container on node {{ "{{" }} $labels.namespace }} restarted"
       {{- end}}
-      {{- if .Values.alerts.highCPU }}
-     - alert: HighCPU
-       expr: round(sum(rate(container_cpu_usage_seconds_total{container=~"(clearblade|cb-.*)",container!~".*exporter", namespace="{{ $.Values.global.deploymentnamespace }}"}[5m])) by (pod, container) / sum(container_spec_cpu_quota{container=~"(clearblade|cb-.*)",container!~".*exporter", namespace="{{ $.Values.global.deploymentnamespace }}"}/container_spec_cpu_period{container=~"(clearblade|cb-.*)",container!~".*exporter", namespace="{{ $.Values.global.deploymentnamespace }}"}) by (pod, container)) > 80
+      {{- if $.Values.alerts.highCPU }}
+     - alert: {{$n.name}}HighCPU
+       expr: round(sum(rate(container_cpu_usage_seconds_total{container=~"(clearblade|cb-.*)",container!~".*exporter", namespace="{{ $n.name }}"}[5m])) by (pod, container) / sum(container_spec_cpu_quota{container=~"(clearblade|cb-.*)",container!~".*exporter", namespace="{{ $n.name }}"}/container_spec_cpu_period{container=~"(clearblade|cb-.*)",container!~".*exporter", namespace="{{ $n.name }}"}) by (pod, container)) > {{ $.Values.alerts.highCPUValue }}
        for: 10m
        annotations:
          summary: "Pod {{ "{{" }} $labels.pod }} cpu is too high: {{ "{{" }} $value }}%"
       {{- end}}
-      {{- if .Values.alerts.fileDescriptors }}
-     - alert: TooManyOpenFileDescriptors
+      {{- if $.Values.alerts.fileDescriptors }}
+     - alert: {{$n.name}}TooManyOpenFileDescriptors
        expr: node_filefd_allocated * on(instance) group_left(nodename) (node_uname_info) > 900000
```

Comment:
Is it worth making the num open descriptors a helm value like you did with `highCPUValue` and `highMemValue`?